### DISCUSSION TITLE
Distance measurement while placing the object (editor)

### DISCUSSION THREAD

**Discussion by rvnexe** [ppy/osu#29364] [Ideas] [answered]
Hi. We can see the distance of the object between the next/previous objects.
![image](https://github.com/user-attachments/assets/a164872b-c41b-4bd8-90cf-f9701e366ea8)

What if we could see it also when placing a new object?
![image](https://github.com/user-attachments/assets/63ab8478-6ed3-4d32-9be0-37167962888b)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/29366
